test/e2e/page-objects/pages/swap/swap-page.ts

```diff
@@ -84,6 +153,59 @@ class SwapPage {
       tag: 'button',
     });
   }
+
+  async checkNoQuotesAvailable(): Promise<void> {
+    await this.driver.waitForSelector({
+      text: `This trade route isn't available right now. Try changing the amount, network, or token and we'll find the best option`,
+      tag: 'p',
```

Comment:
could we define a selector above, and then use the defined selector, instead of hardcoding it here?